ipa_room_exploration/ros/src/coverage_check_server.cpp

```diff
@@ -255,48 +253,16 @@ void coverageCheckServer::drawSeenPoints(cv::Mat& reachable_areas_map, const std
 	// iterate trough all poses and draw them into the given map
 	for(std::vector<geometry_msgs::Pose2D>::const_iterator pose=robot_poses.begin(); pose!=robot_poses.end(); ++pose)
 	{
-//		// get the rotation matrix
```

Comment:
Please keep all the deleted lines in this file until the end of the file. The current setup uses a simplified circular footprint but the commented code uses the real footprint polygon. We did not decide yet, which of these functions will be used in the long run.